beacon_node/network/src/sync/range_sync/batch.rs

```diff
@@ -70,8 +70,11 @@ impl BatchConfig for RangeSyncBatchConfig {
 // Such errors should never be encountered.
 pub struct WrongState(pub(crate) String);
 
-/// Auxiliary type alias for readability.
-type IsFailed = bool;
+/// After batch operations, we use this to communicate whether a batch can continue or not
+pub enum OpOutcome {
```

Comment:
Just a nit but maybe we could have a more complete name for this type?
`BatchOperationOutcome` maybe?

Reply:
done in https://github.com/sigp/lighthouse/pull/3439/commits/6869ba426b57905618e98f18fa757e8a194886dc